docs/spec/stay.en-US.md

```diff
@@ -0,0 +1,102 @@
+
+<img class="preview-img" align="right" alt="good example (special case)" description="If the undo operation is invalid, the Popconfirm is displayed after clicking the delete button. The user can stay on the page to double confirm." src="https://os.alipayobjects.com/rmsportal/hDlhNscVtHdvvgu.png" good>
+
+<img class="preview-img" align="right" alt="bad example" description="
+Abusing the Modal can neither bring the context into the popup, which is prone to interrupt the user’s flow, nor allow the user to undo the change." src="https://os.alipayobjects.com/rmsportal/JEcWKBYlMcYIowX.png" bad>
+
+Double-confirm overlay: Using the Modal to double confirm should be avoided, while affording an opportunity to undo is preferred.
+
+<br>
+
+<img class="preview-img" align="right" alt="example of Detail Overlay " description="Click the eye icon to see more information." src="https://os.alipayobjects.com/rmsportal/WIhhhXExyQBsZwj.png">
+
+Detail Overlay: Allows an overlay to present additional information when the user clicks or hovers over a link or section of content.
+
+
+> Note that when a mouseover event occurs to trigger the Detail Overlay, 0.5-second delay needs to be added, and when the mouse is out, the overlay needs to be closed immediately.
+
+<br>
+
+<img class="preview-img" align="right" alt="example of Input Overlay" description="Click the edit icon to trigger the Input Overlay. Click the outside of it to preserve the input and close it." src="https://os.alipayobjects.com/rmsportal/YehbrRkldqWsezo.png">
+
+Input Overlay: Let the user enter small amounts of text on the overlay.
+
+<br>
+
+---
+
+## Inlays
+
+<img class="preview-img" align="right" alt="example of List Inlay" src="https://os.alipayobjects.com/rmsportal/bHCqDMqXhZvKbve.png">
+
+List Inlay: Works as an effective way to hide detail until needed — while at the same time preserving space on the page for high-level overview information.
+
+<br>
+
+<p><span class="waiting">Detail Inlay (coming soon)</span></p>
+
+<br>
+
+<img class="preview-img" align="right" alt="example of Tabs" src="https://os.alipayobjects.com/rmsportal/MsVyvEIJtlxZWBL.png">
+
+Tabs: Provides additional panels of information accessible by tab controls.
+
+<br>
+
+---
+
+## Virtual Pages
+
+In the process of interaction design, Overlays allow you to bring additional interactions or content in a layer above the cur- rent page. Inlays allow you to do this within the page itself. However, another powerful approach to keeping users engaged on the current page is to create a virtual page. That is to say, we create the illusion of a larger virtual page. 
+
+Virtual Scrolling and Pagination, more on [Patterns／Lists／Show Long Lists](../pattern/list#显示长列表)
+
+Carousel, more on [Patterns／Lists／Show Images](../pattern/list#显示图片)
```

Comment:
/docs/pattern/* have not been translated, so keep it until pattern is translated.